source/containers.tex

```diff
@@ -9200,6 +9238,665 @@
+\indexlibrarymember{emplace_back}{inplace_vector}%
+\begin{itemdecl}
+constexpr reference push_back(const T& x);
+constexpr reference push_back(T&& x);
+template<class... Args>
+  constexpr reference emplace_back(Args&&... args);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{back()}.
+
+\pnum
+\throws
+\tcode{bad_alloc} or
+any exception thrown by initialization of inserted element.
+
+\pnum
+\complexity
+Constant.
+
+\pnum
+\remarks
+If an exception is thrown there are no effects on \tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarymember{try_emplace_back}{inplace_vector}%
+\indexlibrarymember{try_push_back}{inplace_vector}%
+\begin{itemdecl}
+template<class... Args>
+  constexpr pointer try_emplace_back(Args&&... args);
+constexpr pointer try_push_back(const T& x);
+constexpr pointer try_push_back(T&& x);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let \tcode{vals} denote a pack:
+\begin{itemize}
+\item \tcode{std::forward<Args>(args)...} for the first overload,
+\item \tcode{x} for the second overload,
+\item \tcode{set::move(x)} for the third overload.
+\end{itemize}
+
+\pnum
+\expects
+\tcode{T} is \oldconcept{EmplaceConstructible}
+into \tcode{*this} from \tcode{vals...}.
+
+\pnum
+\effects
+If \tcode{size() < capacity()} is \tcode{true},
+appends an object of type \tcode{T}
+direct-non-list-initialized with \tcode{vals...}.
+Otherwise, there are no effects.
+
+\pnum
+\returns
+\keyword{nullptr} if \tcode{size() == capacity()} is \tcode{true},
+otherwise \tcode{addressof(back())}.
+
+\pnum
+\throws
+Nothing unless an exception is thrown by initialization of inserted element.
+
+\pnum
+\complexity
+Constant.
+
+\pnum
+\remarks
+If an exception is thrown there are no effects on \tcode{*this}.
+\end{itemdescr}
+
+\indexlibrarymember{try_append_range}{inplace_vector}%
+\begin{itemdecl}
+template<@\exposconcept{container-compatible-range}@<T> R>
+  constexpr ranges::borrowed_iterator_t<R> try_append_range(R&& rg);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{T} is \oldconcept{EmplaceConstructible}
+from \tcode{*ranges::begin(rg)} into \tcode{*this}.
+
+\pnum
+\effects
+Appends copies of initial elements in \tcode{rg} before \tcode{end()},
+until all elements are inserted or \tcode{size() == capacity()} is \tcode{true}.
+Each iterator in the range \tcode{rg} is dereferenced at most once.
+
+\pnum
+\returns
+%FIXME: Should this be "An iterator past the last inserted element of rg."?
+Iterator past last inserted element of \tcode{rg}.
+
+\pnum
+\complexity
+Linear in the number of elements inserted.
+
+\pnum
+\remarks
+Let $n$ be the value of \tcode{size()} prior to this call.
+If an exception is thrown after insertion of $k$ elements, then
+\tcode{size()} equals $n + k$,
+%FIXME: Are these valid ranges?
+elements in the range \tcode{begin() + \range{0}{$n$}} are not modified, and
+elements in the range \tcode{begin() + \range{$n$}{$n + k$}} correspond to
+the inserted elements.
```

Comment:
```suggestion
If an exception is thrown after the insertion of $k$ elements, then
\tcode{size()} equals $n + k$,
the elements in the range \tcode{[begin(), begin() + n)} are not modified,
and the elements in the range \tcode{[begin() + n, begin() + n + k)} are
copies of the inserted elements.
```

(Perhaps "after the insertion of the $k$th element", or "during the insertion of the $k+1$th element" since that's what we really mean AFAICT?)

Reply:
"the" added.

As for the ranges, I think your rewrite is what was intended (except that the math font and `\range` should be kept), but can we be sure? See same issue above.

As for the wording changes, they're normative - please forward to LWG.